src/theme/CodeBlock/index.tsx

```diff
@@ -17,9 +17,7 @@ export default function CodeBlockWrapper({
         className={clsx("shiki hihi", styles.codeBlock, className)}
         {...props}
       >
-        <div className="code-container line">
-          <code>{children}</code>
-        </div>
```

Comment:
There was a duplication of `.code-container`, and `code`:

![image](https://github.com/LearningTypeScript/site/assets/32631382/d01b6224-7a72-4e8e-b576-7847fdf2536a)